packages/rendermime-interfaces/src/index.ts

```diff
@@ -277,11 +276,6 @@ export namespace IRenderMime {
    * A widget which displays the contents of a mime model.
    */
   export interface IRenderer extends Widget {
-    /**
-     * The application language translator.
-     */
-    translator?: ITranslator;
```

Comment:
If this change is fine to left the package at v3.x, I'm definitely :100: for avoiding downstream mimetype extension to require bumping their package for JLab v4.

I'll raise the question at the next meeting.

Reply:
Sounds good.